projects/api/src/middlewares/validateAuthentication.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response } from 'express'
+import jwt from 'jsonwebtoken'
+import { ServerLogger, logLevels } from '../utils'
+import { secretKey } from '../constants/secret'
+
+interface IAuthenticatedRequest extends Request {
+    user?: string | object

```

Comment:
Why it can be a string or an object?

Reply:
`jwt.verify(token, secretKey)` returns either a string or an object